internal/errors/resolver/git.go

```diff
@@ -105,7 +105,6 @@ func (*gitExecErrorResolver) Resolve(err error) (ResolvedResult, bool) {
 		msg = ExecuteTemplate(genericGitExecError, tmplArgs)
 	}
 	return ResolvedResult{
```

Comment:
The struct name here is `ResolvedResult` and not `ResolvedError` which sounds like it is a generic Result struct and not Error struct. Is it justified to have default 1 ?

Reply:
yeah, exit code 1 is a catchall for general errors. So unless something else is specified, we should just return error code 1 on errors.

I agree with your comment that `ResolvedResult` might be a little misleading. But lets rename that in a separate PR.